src/instruments/src/PFD/SpeedIndicator.tsx

```diff
@@ -523,39 +495,35 @@ class SpeedTrendArrow extends DisplayComponent<{ airspeed: Subscribable<number>,
 }
 
 class VLsBar extends DisplayComponent<{ bus: EventBus }> {
-    private previousTime = (new Date() as any).appTime();
-
     private vlsPath = Subject.create<string>('');
 
+    private refElement = FSComponent.createRef<SVGGElement>();
+
     private vAlphaProt = new Arinc429Word(0);
 
     private vStallWarn = new Arinc429Word(0);
 
     private airSpeed= new Arinc429Word(0);
 
-    private vls= 0;
+    private vls= new Arinc429Word(0);
 
     private fcdc1DiscreteWord1 = new Arinc429Word(0);
 
     private fcdc2DiscreteWord1 = new Arinc429Word(0);
 
-    private smoothSpeeds = (vlsDestination: number) => {
-        const currentTime = (new Date() as any).appTime();
-        const deltaTime = currentTime - this.previousTime;
-
-        const seconds = deltaTime / 1000;
-        const vls = SmoothSin(this.vls, vlsDestination, 0.5, seconds);
-        this.previousTime = currentTime;
-        return vls;
-    };
-
     private setVlsPath() {
-        const normalLawActive = this.fcdc1DiscreteWord1.getBitValueOr(11, false) || this.fcdc2DiscreteWord1.getBitValueOr(11, false);
+        if (this.vls.isNormalOperation()) {
+            this.refElement.instance.style.visibility = 'visible';
```

Comment:
I believe it is more efficient to use as subject (e.g. `Subject.create('visible')`) and use that in [Line 566](https://github.com/flybywiresim/a32nx/pull/7512/files#diff-258e500ddb10dce616498c2ec1b32b7637850e32a26739a176761f7dbe9f794bR566) as a visibility toggle. This way the DOM only gets modified when there are actual changes. I know it's been done the way you did in in many places, but I didn't know that the subject way is better back then. You can also keep it for now as I planned to make another pass with the PFD to change it in the other places as well.